01_fib/choohy/main_test.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestFibNeg11(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	fib(-11)
+
+	expected := strconv.Quote(``)
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Helper()
+		t.Errorf(expected)
+		t.Errorf(actual)
+	}
+}
+func TestFib0(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	fib(0)
+
+	expected := strconv.Quote(``)
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Helper()
+		t.Errorf(expected)
+		t.Errorf(actual)
+	}
+}
+func TestFib1(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	fib(1)
+
+	expected := strconv.Quote(`1
+`)
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Helper()
+		t.Errorf(expected)
```

Comment:
You can combine these into a single call, there isn't a need to call this twice.
```go
t.Errorf("Expected: %v, Actual: %v", expected, actual)
```